packages/cli-template-contracts-hardhat/README.md

```diff
@@ -4,13 +4,19 @@ This project demonstrates a basic Semaphore use case. It comes with a sample con
 
 ## Usage
 
-### Compile
+### Install dependencies
+
+```bash
+yarn
+```
```

Comment:
This is usually another `## Install` section in the other packages, right before the `## Usage` one.

e.g.

```md
## Install

Install the dependencies:

`` `bash
yarn
`` `

## Usage

...
```